pkg/controller/service/service_controller.go

```diff
@@ -60,13 +60,19 @@ const (
 	minRetryDelay = 5 * time.Second
 	maxRetryDelay = 300 * time.Second
 
-	// LabelNodeRoleMaster specifies that a node is a master
-	// It's copied over to kubeadm until it's merged in core: https://github.com/kubernetes/kubernetes/pull/39112
+	// LabelNodeRoleMaster specifies that a node is a master. This label is deprecated and
+	// only considered when the LegacyNodeRoleBehavior feature gate is on.
 	LabelNodeRoleMaster = "node-role.kubernetes.io/master"
```

Comment:
can we make this private or inline to avoid picking up new references?

Reply:
already on the way out